.github/workflows/RequestReview.yml

```diff
@@ -73,23 +75,26 @@ jobs:
             }
             //
             async function processIssue(issue) {
-                removeAssignees(issue);
-                addAssignee(issue, context.payload.requested_reviewer.login);
-                const card = await findCard(issue.url);
-                if (card) {
-                    console.log("Moving card");
-                    github.projects.moveCard({ card_id: card.id, position: "bottom", column_id: REVIEW_IN_PROGRESS_COLUMN });
-                } else if (issue.pull_request) {
-                    console.log("Creating PR card");
-                    github.projects.createCard({ column_id: REVIEW_IN_PROGRESS_COLUMN, content_id: context.payload.pull_request.id, content_type: "PullRequest" });
-                } else {
-                    console.log("Creating Issue card");
-                    github.projects.createCard({ column_id: REVIEW_IN_PROGRESS_COLUMN, content_id: issue.id, content_type: "Issue" });
+                if(issue.state == "open") {
+                  removeAssignees(issue);
```

Comment:
if there is a 2nd reviewer added, I believe we shouldn't reset+add, but add another face on the kanban, to reflect the people that should have a look

Reply:
Let's stick with baby steps here as well. Review state is very hard to read from API. I can't get list of reviewers with "yellow dot in UI".

Reply:
I added a note about this idea to my Kanban backlog.

Reply:
sure, it's an edge case, if it's not easy to fix, we leave it for later